File: bilibili/src/main/java/com/bilibili/ui/test/fragment/PlaceHolderFragment.java
package com.bilibili.ui.test.fragment;

import android.widget.TextView;

import com.bilibili.R;
import com.common.base.BaseFragment;

import java.util.Random;

import butterknife.BindView;

/**
 * Created by miserydx on 17/12/19.
 */

public class PlaceHolderFragment extends BaseFragment {

    @BindView(R.id.tv_place_holder)
    TextView tvPlaceHolder;

    @Override
    protected int getLayoutId() {
        return R.layout.fragment_placeholder;
    }

    @Override
    protected void initInject() {

    }

    @Override
    protected void initViewAndEvent() {
        tvPlaceHolder.setText("敬请期待");
    }
}

File: bilibili/src/main/java/com/bilibili/ui/test/fragment/PlaceHolderFragment.kt
package com.bilibili.ui.test.fragment

import android.widget.TextView
import butterknife.BindView
import com.bilibili.R
import com.common.base.BaseFragment
import java.util.*

/**
 * Created by miserydx on 17/10/26.
 */
class PlaceHolderFragment : BaseFragment() {

    @BindView(R.id.tv_place_holder)
    lateinit var tvPlaceHolder: TextView

    override fun getLayoutId(): Int = R.layout.fragment_placeholder

    override fun initInject() {

    }

    override fun initViewAndEvent() {
        tvPlaceHolder.text = Random().nextInt(2345).toString()
    }
}